pom.xml

```diff
@@ -926,7 +932,13 @@
             <forkCount>1</forkCount>
             <reuseForks>false</reuseForks>
             <forkedProcessTimeoutInSeconds>900</forkedProcessTimeoutInSeconds>
-            <argLine>-Xmx1024m -XX:+HeapDumpOnOutOfMemoryError</argLine>
+            <argLine>-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError</argLine>
+            <argLine>
+              --add-opens=java.base/java.lang=ALL-UNNAMED
+              --add-opens=java.base/java.util=ALL-UNNAMED
```

Comment:
Consolidation of all "--add-opens" in a single variable and refer to it in other module pom would be better.